src/controls/Button.jsx

```diff
@@ -35,60 +36,53 @@ const buttonStyles = {
  * @param {Object} [props.style] Desired JSX-formatted styling object; overrides all other style attributes
  * @returns {import("react").ReactComponentElement}
  */
-export default function Button(props) {
-  let {
-    text,
-    onClick,
-    width,
-    type,
-    size,
-    height,
-    hoverStart,
-    backgroundColor,
-    activeBackgroundColor,
-    style,
-  } = props;
+const Button = (props) => {
```

Comment:
```suggestion
export default function Button(props) {
```
`Button` is a full-fledged React 16.8+ hook-based component, so this declaration style better matches industry standards